<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Guerrilla Analytics | Data science for Biology - Portfolio Jetske Moot</title>
  <meta name="description" content="5 Guerrilla Analytics | Data science for Biology - Portfolio Jetske Moot" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Guerrilla Analytics | Data science for Biology - Portfolio Jetske Moot" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Guerrilla Analytics | Data science for Biology - Portfolio Jetske Moot" />
  
  
  

<meta name="author" content="Jetske Moot" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="structurele-eiwitanalyse-met-bio3d.html"/>
<link rel="next" href="peer-review.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/r3dmol-1.6.2/3Dmol-nojquery-min.js"></script>
<script src="libs/r3dmol-binding-0.1.2/r3dmol.js"></script>
<script src="libs/plotly-binding-4.10.4/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introductie</a></li>
<li class="chapter" data-level="2" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i><b>2</b> Curriculum vitae</a>
<ul>
<li class="chapter" data-level="2.1" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#over-mij"><i class="fa fa-check"></i><b>2.1</b> Over mij</a></li>
<li class="chapter" data-level="2.2" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#opleidingen"><i class="fa fa-check"></i><b>2.2</b> Opleidingen</a></li>
<li class="chapter" data-level="2.3" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#projecten"><i class="fa fa-check"></i><b>2.3</b> Projecten</a></li>
<li class="chapter" data-level="2.4" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#werkervaring"><i class="fa fa-check"></i><b>2.4</b> Werkervaring</a></li>
<li class="chapter" data-level="2.5" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#vaardigheden"><i class="fa fa-check"></i><b>2.5</b> Vaardigheden</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#laboratoriumvaardigheden"><i class="fa fa-check"></i><b>2.5.1</b> Laboratoriumvaardigheden</a></li>
<li class="chapter" data-level="2.5.2" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#data-analyse-vaardigheden"><i class="fa fa-check"></i><b>2.5.2</b> Data-analyse vaardigheden</a></li>
<li class="chapter" data-level="2.5.3" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html#talen"><i class="fa fa-check"></i><b>2.5.3</b> Talen</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="plan-voor-de-toekomst.html"><a href="plan-voor-de-toekomst.html"><i class="fa fa-check"></i><b>3</b> Plan voor de toekomst</a>
<ul>
<li class="chapter" data-level="3.1" data-path="plan-voor-de-toekomst.html"><a href="plan-voor-de-toekomst.html#toekomstbeeld-en-huidig-beeld"><i class="fa fa-check"></i><b>3.1</b> Toekomstbeeld en huidig beeld</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html"><i class="fa fa-check"></i><b>4</b> Structurele eiwitanalyse met Bio3D</a>
<ul>
<li class="chapter" data-level="4.1" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#mijn-eerste-ervaring-met-alphafold"><i class="fa fa-check"></i><b>4.1</b> Mijn eerste ervaring met AlphaFold</a></li>
<li class="chapter" data-level="4.2" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#introductie-bio3d"><i class="fa fa-check"></i><b>4.2</b> Introductie Bio3D</a></li>
<li class="chapter" data-level="4.3" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#planning"><i class="fa fa-check"></i><b>4.3</b> Planning</a></li>
<li class="chapter" data-level="4.4" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#beginning-structure-analysis"><i class="fa fa-check"></i><b>4.4</b> Beginning Structure Analysis</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#tutorial"><i class="fa fa-check"></i><b>4.4.1</b> Tutorial</a></li>
<li class="chapter" data-level="4.4.2" data-path="structurele-eiwitanalyse-met-bio3d.html"><a href="structurele-eiwitanalyse-met-bio3d.html#rmsd-en-visualisatie-op-eigen-datasets"><i class="fa fa-check"></i><b>4.4.2</b> RMSD en visualisatie op eigen datasets</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="guerrilla-analytics.html"><a href="guerrilla-analytics.html"><i class="fa fa-check"></i><b>5</b> Guerrilla Analytics</a>
<ul>
<li class="chapter" data-level="5.1" data-path="guerrilla-analytics.html"><a href="guerrilla-analytics.html#daur2"><i class="fa fa-check"></i><b>5.1</b> DAUR2</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="guerrilla-analytics.html"><a href="guerrilla-analytics.html#readme-file"><i class="fa fa-check"></i><b>5.1.1</b> README file</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="guerrilla-analytics.html"><a href="guerrilla-analytics.html#portfolio"><i class="fa fa-check"></i><b>5.2</b> Portfolio</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="guerrilla-analytics.html"><a href="guerrilla-analytics.html#readme-file-1"><i class="fa fa-check"></i><b>5.2.1</b> README file</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="peer-review.html"><a href="peer-review.html"><i class="fa fa-check"></i><b>6</b> Peer review</a>
<ul>
<li class="chapter" data-level="6.1" data-path="peer-review.html"><a href="peer-review.html#data-analyse-van-c.-elegans-blootstelling"><i class="fa fa-check"></i><b>6.1</b> Data analyse van C. elegans blootstelling</a></li>
<li class="chapter" data-level="6.2" data-path="peer-review.html"><a href="peer-review.html#reproduceerbaarheid-van-wetenschappelijk-onderzoek"><i class="fa fa-check"></i><b>6.2</b> Reproduceerbaarheid van wetenschappelijk onderzoek</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="peer-review.html"><a href="peer-review.html#beoordeling-van-onderzoek-op-reproduceerbaarheid"><i class="fa fa-check"></i><b>6.2.1</b> Beoordeling van onderzoek op reproduceerbaarheid</a></li>
<li class="chapter" data-level="6.2.2" data-path="peer-review.html"><a href="peer-review.html#zelf-de-code-reproduceren"><i class="fa fa-check"></i><b>6.2.2</b> Zelf de code reproduceren</a></li>
<li class="chapter" data-level="6.2.3" data-path="peer-review.html"><a href="peer-review.html#referentie"><i class="fa fa-check"></i><b>6.2.3</b> Referentie</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="r-package.html"><a href="r-package.html"><i class="fa fa-check"></i><b>7</b> R package</a></li>
<li class="chapter" data-level="8" data-path="projecticum-alternative-splicing-scrna-seq.html"><a href="projecticum-alternative-splicing-scrna-seq.html"><i class="fa fa-check"></i><b>8</b> Projecticum alternative splicing scRNA-seq</a></li>
<li class="chapter" data-level="9" data-path="ecdc-covid-19-geparametriseerd-rapport.html"><a href="ecdc-covid-19-geparametriseerd-rapport.html"><i class="fa fa-check"></i><b>9</b> ECDC COVID-19 geparametriseerd rapport</a>
<ul>
<li class="chapter" data-level="9.1" data-path="ecdc-covid-19-geparametriseerd-rapport.html"><a href="ecdc-covid-19-geparametriseerd-rapport.html#yaml-header-van-geparametriseerd-covid-19-rapport"><i class="fa fa-check"></i><b>9.1</b> YAML header van geparametriseerd COVID-19 rapport</a></li>
<li class="chapter" data-level="9.2" data-path="ecdc-covid-19-geparametriseerd-rapport.html"><a href="ecdc-covid-19-geparametriseerd-rapport.html#codes-in-geparametriseerd-covid-19-rapport"><i class="fa fa-check"></i><b>9.2</b> Codes in geparametriseerd COVID-19 rapport</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data science for Biology - Portfolio Jetske Moot</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="guerrilla-analytics" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">5</span> Guerrilla Analytics<a href="guerrilla-analytics.html#guerrilla-analytics" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In dit deel van mijn portfolio laat ik zien hoe ik de Guerilla Analytics richtlijnen toepas op data.</p>
<div id="daur2" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> DAUR2<a href="guerrilla-analytics.html#daur2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Hier laat ik mijn data management zien op de data voor de eindoprdacht van de cursus DAUR2 afkomstig uit Data Science for Biology 1.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="guerrilla-analytics.html#cb35-1" tabindex="-1"></a><span class="fu">library</span>(fs)</span>
<span id="cb35-2"><a href="guerrilla-analytics.html#cb35-2" tabindex="-1"></a>fs<span class="sc">::</span><span class="fu">dir_tree</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;daur2_eindopdracht&quot;</span>))</span></code></pre></div>
<pre><code>## /Users/macbook12/dsfb2_2025_jm/dsfb2_workflows_portfolio_jm/daur2_eindopdracht
## ├── README.md
## ├── data
## │   ├── bam
## │   │   ├── SRR7866699.bam
## │   │   ├── SRR7866699.bam.indel.vcf
## │   │   ├── SRR7866699.bam.summary
## │   │   ├── SRR7866700.bam
## │   │   ├── SRR7866700.bam.indel.vcf
## │   │   ├── SRR7866700.bam.summary
## │   │   ├── SRR7866705.bam
## │   │   ├── SRR7866705.bam.indel.vcf
## │   │   ├── SRR7866705.bam.summary
## │   │   ├── SRR7866706.bam
## │   │   ├── SRR7866706.bam.indel.vcf
## │   │   └── SRR7866706.bam.summary
## │   ├── counts
## │   │   └── read_counts_OC3.rds
## │   └── fastqc
## │       ├── SRR7866699_1_fastqc.html
## │       ├── SRR7866699_1_fastqc.zip
## │       ├── SRR7866699_2_fastqc.html
## │       ├── SRR7866699_2_fastqc.zip
## │       ├── SRR7866700_1_fastqc.html
## │       ├── SRR7866700_1_fastqc.zip
## │       ├── SRR7866700_2_fastqc.html
## │       ├── SRR7866700_2_fastqc.zip
## │       ├── SRR7866705_1_fastqc.html
## │       ├── SRR7866705_1_fastqc.zip
## │       ├── SRR7866705_2_fastqc.html
## │       ├── SRR7866705_2_fastqc.zip
## │       ├── SRR7866706_1_fastqc.html
## │       ├── SRR7866706_1_fastqc.zip
## │       ├── SRR7866706_2_fastqc.html
## │       └── SRR7866706_2_fastqc.zip
## ├── daur2_eindopdracht.Rmd
## └── raw_data
##     ├── fastq
##     │   ├── SRR7866699_1.fastq.gz
##     │   ├── SRR7866699_2.fastq.gz
##     │   ├── SRR7866700_1.fastq.gz
##     │   ├── SRR7866700_2.fastq.gz
##     │   ├── SRR7866705_1.fastq.gz
##     │   ├── SRR7866705_2.fastq.gz
##     │   ├── SRR7866706_1.fastq.gz
##     │   └── SRR7866706_2.fastq.gz
##     └── onecut_sampledata_OC3.csv</code></pre>
<div id="readme-file" class="section level3 hasAnchor" number="5.1.1">
<h3><span class="header-section-number">5.1.1</span> README file<a href="guerrilla-analytics.html#readme-file" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Dit is de inhoud van de README file voor de data management voor de eindoprdacht van de cursus DAUR2.</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="guerrilla-analytics.html#cb37-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">readLines</span>(<span class="st">&quot;daur2_eindopdracht/README.md&quot;</span>), <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>)</span></code></pre></div>
<pre><code>## *********************
## 
## PROJECT INFORMATIE
## 
## *********************
## 
## **Titel**              
## RNA-seq analyse van transdifferentiatie: ONECUT3 versus BCL2 in humane fibroblasten        
## 
## **Auteur informatie**        
## *Principal Investigator*         
## Naam: Timothy E. Reddy         
## Instituut: Duke University      
##         
## *Co-Investigator*        
## Naam: Jetske Moot        
## Instituut: Hogeschool Utrecht        
##         
## *Datum van data collectie*        
## Data gepubliceerd in 2019, verzameld in 2018        
## *Funder*        
## National Institutes of Health (NIH)        
## Grant ID: GM118650     
##         
## *************************        
## 
## DATA TOEGANG INFORMATIE
## 
## *************************
## 
## **Licenties / beperkingen voor toegang tot de dataset**         
## Vrij toegankelijk via NCBI GEO (open access)        
## 
## **Links naar publicaties die de data gebruiken**        
## Black et al. (2019), Nucleic Acids Research        
## DOI: https://doi.org/10.1093/nar/gkz224        
## 
## **Links naar andere publiek toegankelijke locaties van de data**        
## GEO: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122350        
## *SRA Run IDs*        
## SRR7866699 (BCL2 replicate 1)        
## SRR7866700 (BCL2 replicate 2)        
## SRR7866705 (ONECUT3 replicate 1)        
## SRR7866706 (ONECUT3 replicate 2)        
##         
## ****************************
## 
## METHODEN VOOR DATA VERZAMELING
## 
## ****************************
## 
## **Methoden voor data verzameling**        
## Menselijke fibroblasten werden getransduceerd met dCas9-VPR en specifieke gRNAs gericht op transcriptiefactoren. Voor deze dataset werd óf BCL2 óf ONECUT3 endogeen geactiveerd. 5 dagen na transductie werd RNA geïsoleerd en gesequencet.       
## 
## **Data processing methoden**        
## Kwaliteitscontrole met FastQC       
## Alignments met STAR aligner       
## Feature quantificatie met featureCounts       
## Analyse uitgevoerd in R met DESeq2       
## Analyse opgenomen in eindopdracht_daur2.Rmd       
##        
## **Instrument informatie**       
## Illumina HiSeq 4000       
## Stranded paired-end sequencing (2 × 100 bp)       
##        
## **Quality assurance procedures**       
## FastQC output beschikbaar in de directory /home/dsfb2_2025_jm/dsfb2_workflows_portfolio_jm/daur2_eindopdracht/data/fastqc
##        
## **Analyse methoden**       
## DESeq2 voor normalisatie en DGE       
## PCA voor sample clustering       
## Volcano plots en heatmaps       
## GO-enrichment analysis       
##        
## ************************
## 
## SAMENVATTING VAN DATA BESTANDEN
## 
## ************************
## 
## **Lijst van data bestanden**
## 
## File naam                      Korte omschrijving
## ----------------------------------------------
## onecut_sampledata_OC3.csv      Sample metadata (behandeling, ID&#39;s)        
## read_counts_OC3.rds            R object met ruwe count matrix        
## eindopdracht_daur2.Rmd         Analyse- en visualisatie        
## SRR7866699_1.fastq.gz          Paired-end forward reads voor sample 1        
## SRR7866699_2.fastq.gz          Paired-end reverse reads voor sample 1        
## SRR7866700_1.fastq.gz          Paired-end forward reads voor sample 2        
## SRR7866700_2.fastq.gz          Paired-end reverse reads voor sample 2        
## SRR7866705_1.fastq.gz          Paired-end forward reads voor sample 3        
## SRR7866705_2.fastq.gz          Paired-end reverse reads voor sample 3        
## SRR7866706_1.fastq.gz          Paired-end forward reads voor sample 4        
## SRR7866706_2.fastq.gz          Paired-end reverse reads voor sample 4        
## </code></pre>
</div>
</div>
<div id="portfolio" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Portfolio<a href="guerrilla-analytics.html#portfolio" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Hier laat ik mijn data management zien op de data van mijn portfolio voor Data Science for Biology 2.</p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="guerrilla-analytics.html#cb39-1" tabindex="-1"></a>fs<span class="sc">::</span><span class="fu">dir_tree</span>(here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;../dsfb2_workflows_portfolio_jm&quot;</span>))</span></code></pre></div>
<pre><code>## /Users/macbook12/dsfb2_2025_jm/dsfb2_workflows_portfolio_jm/../dsfb2_workflows_portfolio_jm
## ├── 01_cv_jm.Rmd
## ├── 02_toekomst_jm.Rmd
## ├── 03_vrije_opdracht_jm.Rmd
## ├── 04_guerrilla_analytics.Rmd
## ├── 05_reproducible.Rmd
## ├── 06_rpackage.Rmd
## ├── 07_projecticum.Rmd
## ├── 08_covid19.Rmd
## ├── LICENSE
## ├── README.md
## ├── _book
## │   ├── 404.html
## │   ├── _main_files
## │   │   └── figure-html
## │   │       ├── threshold plotten-1.png
## │   │       ├── unnamed-chunk-10-1.png
## │   │       ├── unnamed-chunk-11-1.png
## │   │       ├── unnamed-chunk-12-1.png
## │   │       ├── unnamed-chunk-13-1.png
## │   │       ├── unnamed-chunk-14-1.png
## │   │       ├── unnamed-chunk-17-1.png
## │   │       ├── unnamed-chunk-17-2.png
## │   │       ├── unnamed-chunk-21-1.png
## │   │       ├── unnamed-chunk-22-1.png
## │   │       ├── unnamed-chunk-23-1.png
## │   │       ├── unnamed-chunk-24-1.png
## │   │       ├── unnamed-chunk-25-1.png
## │   │       ├── unnamed-chunk-28-1.png
## │   │       ├── unnamed-chunk-29-1.png
## │   │       ├── unnamed-chunk-31-1.png
## │   │       ├── unnamed-chunk-35-1.png
## │   │       ├── unnamed-chunk-36-1.png
## │   │       ├── unnamed-chunk-37-1.png
## │   │       ├── unnamed-chunk-38-1.png
## │   │       ├── unnamed-chunk-40-1.png
## │   │       ├── unnamed-chunk-41-1.png
## │   │       ├── unnamed-chunk-42-1.png
## │   │       ├── unnamed-chunk-43-1.png
## │   │       ├── unnamed-chunk-44-1.png
## │   │       ├── unnamed-chunk-45-1.png
## │   │       ├── unnamed-chunk-46-1.png
## │   │       ├── unnamed-chunk-47-1.png
## │   │       ├── unnamed-chunk-8-1.png
## │   │       └── unnamed-chunk-9-1.png
## │   ├── curriculum-vitae.html
## │   ├── cv.html
## │   ├── ecdc-covid-19-geparametriseerd-rapport.html
## │   ├── guerrilla-analytics.html
## │   ├── images
## │   │   ├── alphafold_example.png
## │   │   └── bio3d_example.png
## │   ├── index.html
## │   ├── introduction.html
## │   ├── leren-van-alphafold---nauwkeurige-eiwitstructuur-voorspelling.html
## │   ├── leren-van-bio3d---structurele-eiwitanalyse.html
## │   ├── libs
## │   │   ├── anchor-sections-1.1.0
## │   │   │   ├── anchor-sections-hash.css
## │   │   │   ├── anchor-sections.css
## │   │   │   └── anchor-sections.js
## │   │   ├── crosstalk-1.2.1
## │   │   │   ├── css
## │   │   │   │   └── crosstalk.min.css
## │   │   │   └── js
## │   │   │       └── crosstalk.min.js
## │   │   ├── gitbook-2.6.7
## │   │   │   ├── css
## │   │   │   │   ├── fontawesome
## │   │   │   │   │   └── fontawesome-webfont.ttf
## │   │   │   │   ├── plugin-bookdown.css
## │   │   │   │   ├── plugin-clipboard.css
## │   │   │   │   ├── plugin-fontsettings.css
## │   │   │   │   ├── plugin-highlight.css
## │   │   │   │   ├── plugin-search.css
## │   │   │   │   ├── plugin-table.css
## │   │   │   │   └── style.css
## │   │   │   └── js
## │   │   │       ├── app.min.js
## │   │   │       ├── clipboard.min.js
## │   │   │       ├── jquery.highlight.js
## │   │   │       ├── plugin-bookdown.js
## │   │   │       ├── plugin-clipboard.js
## │   │   │       ├── plugin-fontsettings.js
## │   │   │       ├── plugin-search.js
## │   │   │       └── plugin-sharing.js
## │   │   ├── htmltools-fill-0.5.8.1
## │   │   │   └── fill.css
## │   │   ├── htmlwidgets-1.6.4
## │   │   │   └── htmlwidgets.js
## │   │   ├── jquery-3.6.0
## │   │   │   └── jquery-3.6.0.min.js
## │   │   ├── plotly-binding-4.10.4
## │   │   │   └── plotly.js
## │   │   ├── plotly-htmlwidgets-css-2.11.1
## │   │   │   └── plotly-htmlwidgets.css
## │   │   ├── plotly-main-2.11.1
## │   │   │   └── plotly-latest.min.js
## │   │   ├── r3dmol-1.6.2
## │   │   │   └── 3Dmol-nojquery-min.js
## │   │   ├── r3dmol-binding-0.1.2
## │   │   │   └── r3dmol.js
## │   │   └── typedarray-0.1
## │   │       └── typedarray.min.js
## │   ├── main.html
## │   ├── peer-review.html
## │   ├── plan-voor-de-toekomst.html
## │   ├── projecticum-alternative-splicing-scrna-seq.html
## │   ├── r-package.html
## │   ├── reproducible-science.html
## │   ├── search_index.json
## │   ├── structurele-eiwitanalyse-met-bio3d.html
## │   └── vrije-opdracht---titel-nader-te-bepalen.html
## ├── _bookdown_files
## ├── _main.Rmd
## ├── _main_files
## │   └── figure-html
## │       ├── threshold plotten-1.png
## │       ├── unnamed-chunk-10-1.png
## │       ├── unnamed-chunk-11-1.png
## │       ├── unnamed-chunk-12-1.png
## │       ├── unnamed-chunk-13-1.png
## │       ├── unnamed-chunk-14-1.png
## │       ├── unnamed-chunk-17-1.png
## │       ├── unnamed-chunk-17-2.png
## │       ├── unnamed-chunk-21-1.png
## │       ├── unnamed-chunk-22-1.png
## │       ├── unnamed-chunk-23-1.png
## │       ├── unnamed-chunk-24-1.png
## │       ├── unnamed-chunk-25-1.png
## │       ├── unnamed-chunk-28-1.png
## │       ├── unnamed-chunk-29-1.png
## │       ├── unnamed-chunk-31-1.png
## │       ├── unnamed-chunk-35-1.png
## │       ├── unnamed-chunk-36-1.png
## │       ├── unnamed-chunk-37-1.png
## │       ├── unnamed-chunk-38-1.png
## │       ├── unnamed-chunk-40-1.png
## │       ├── unnamed-chunk-41-1.png
## │       ├── unnamed-chunk-42-1.png
## │       ├── unnamed-chunk-43-1.png
## │       ├── unnamed-chunk-44-1.png
## │       ├── unnamed-chunk-45-1.png
## │       ├── unnamed-chunk-46-1.png
## │       ├── unnamed-chunk-47-1.png
## │       ├── unnamed-chunk-8-1.png
## │       └── unnamed-chunk-9-1.png
## ├── bibliography.json
## ├── daur2_eindopdracht
## │   ├── README.md
## │   ├── data
## │   │   ├── bam
## │   │   │   ├── SRR7866699.bam
## │   │   │   ├── SRR7866699.bam.indel.vcf
## │   │   │   ├── SRR7866699.bam.summary
## │   │   │   ├── SRR7866700.bam
## │   │   │   ├── SRR7866700.bam.indel.vcf
## │   │   │   ├── SRR7866700.bam.summary
## │   │   │   ├── SRR7866705.bam
## │   │   │   ├── SRR7866705.bam.indel.vcf
## │   │   │   ├── SRR7866705.bam.summary
## │   │   │   ├── SRR7866706.bam
## │   │   │   ├── SRR7866706.bam.indel.vcf
## │   │   │   └── SRR7866706.bam.summary
## │   │   ├── counts
## │   │   │   └── read_counts_OC3.rds
## │   │   └── fastqc
## │   │       ├── SRR7866699_1_fastqc.html
## │   │       ├── SRR7866699_1_fastqc.zip
## │   │       ├── SRR7866699_2_fastqc.html
## │   │       ├── SRR7866699_2_fastqc.zip
## │   │       ├── SRR7866700_1_fastqc.html
## │   │       ├── SRR7866700_1_fastqc.zip
## │   │       ├── SRR7866700_2_fastqc.html
## │   │       ├── SRR7866700_2_fastqc.zip
## │   │       ├── SRR7866705_1_fastqc.html
## │   │       ├── SRR7866705_1_fastqc.zip
## │   │       ├── SRR7866705_2_fastqc.html
## │   │       ├── SRR7866705_2_fastqc.zip
## │   │       ├── SRR7866706_1_fastqc.html
## │   │       ├── SRR7866706_1_fastqc.zip
## │   │       ├── SRR7866706_2_fastqc.html
## │   │       └── SRR7866706_2_fastqc.zip
## │   ├── daur2_eindopdracht.Rmd
## │   └── raw_data
## │       ├── fastq
## │       │   ├── SRR7866699_1.fastq.gz
## │       │   ├── SRR7866699_2.fastq.gz
## │       │   ├── SRR7866700_1.fastq.gz
## │       │   ├── SRR7866700_2.fastq.gz
## │       │   ├── SRR7866705_1.fastq.gz
## │       │   ├── SRR7866705_2.fastq.gz
## │       │   ├── SRR7866706_1.fastq.gz
## │       │   └── SRR7866706_2.fastq.gz
## │       └── onecut_sampledata_OC3.csv
## ├── dsfb2_workflows_portfolio_jm.Rproj
## ├── images
## ├── index.Rmd
## ├── protein_analysis
## │   ├── 1AGR.pdb
## │   ├── 1FQJ.pdb
## │   ├── 1GG2.pdb
## │   ├── 1KJY.pdb
## │   ├── 1TAG.pdb
## │   ├── 1TND.pdb
## │   ├── 3l9p.pdb
## │   ├── 4fnx.pdb
## │   ├── 4obe.pdb
## │   ├── 4tq9.pdb
## │   ├── aln.fa
## │   ├── data
## │   │   ├── 3l9p_aligned.pdb
## │   │   ├── 4fnx_aligned.pdb
## │   │   ├── 4obeA_aligned.pdb
## │   │   └── 4tq9A_aligned.pdb
## │   ├── own_protein_analysis.Rmd
## │   ├── protein_analysis_tutorial.Rmd
## │   └── split_chain
## │       ├── 1AGR_A.pdb
## │       ├── 1FQJ_A.pdb
## │       ├── 1GG2_A.pdb
## │       ├── 1KJY_A.pdb
## │       ├── 1TAG_A.pdb
## │       ├── 1TND_B.pdb
## │       ├── 3l9p_A.pdb
## │       ├── 4fnx_A.pdb
## │       ├── 4obe_A.pdb
## │       └── 4tq9_A.pdb
## ├── raw_data
## │   └── CE.LIQ.FLOW.062_Tidydata.xlsx
## └── repro_opdracht
##     ├── README.txt
##     ├── analysis
##     │   ├── analysis_data
##     │   │   ├── af_edc_clinvar.tsv
##     │   │   ├── af_foldx_clinvar.tsv.gz
##     │   │   ├── clinvar_all_uniprot_2023-06-03.tsv.gz
##     │   │   ├── clinvar_gnomad_veps.tsv.gz
##     │   │   ├── ddg2p_monoallelic_uniprot.txt
##     │   │   ├── gene_to_uniprot_2023_02.tsv
##     │   │   ├── huang_hi_predictions.tsv
##     │   │   ├── omim_inheritance_2023-05-25.tsv
##     │   │   ├── shihab_hi_predictions.tsv
##     │   │   └── steinberg_hi_predictions.tsv
##     │   ├── model_analysis.Rmd
##     │   ├── panther
##     │   │   ├── dn_list.txt
##     │   │   ├── gof_list.txt
##     │   │   ├── panther_enrichment_dn.txt
##     │   │   └── panther_enrichment_gof.txt
##     │   ├── predictions
##     │   │   ├── pdn_svm_poly_2023-07-25.tsv
##     │   │   ├── pgof_svm_poly_2023-07-25.tsv
##     │   │   └── plof_svm_poly_2023-07-28.tsv
##     │   ├── test_set
##     │   │   ├── dn_test_pred.tsv
##     │   │   ├── gof_test_pred.tsv
##     │   │   └── lof_test_pred.tsv
##     │   └── thresholds
##     │       ├── dn_thresholds.tsv
##     │       ├── gof_thresholds.tsv
##     │       └── lof_thresholds.tsv
##     ├── colab
##     │   ├── fx
##     │   │   ├── ddg_ranks.tsv
##     │   │   ├── up_1.tsv
##     │   │   ├── up_10.tsv
##     │   │   ├── up_11.tsv
##     │   │   ├── up_12.tsv
##     │   │   ├── up_13.tsv
##     │   │   ├── up_14.tsv
##     │   │   ├── up_15.tsv
##     │   │   ├── up_16.tsv
##     │   │   ├── up_17.tsv
##     │   │   ├── up_18.tsv
##     │   │   ├── up_19.tsv
##     │   │   ├── up_2.tsv
##     │   │   ├── up_20.tsv
##     │   │   ├── up_21.tsv
##     │   │   ├── up_22.tsv
##     │   │   ├── up_23.tsv
##     │   │   ├── up_24.tsv
##     │   │   ├── up_25.tsv
##     │   │   ├── up_26.tsv
##     │   │   ├── up_27.tsv
##     │   │   ├── up_28.tsv
##     │   │   ├── up_29.tsv
##     │   │   ├── up_3.tsv
##     │   │   ├── up_30.tsv
##     │   │   ├── up_31.tsv
##     │   │   ├── up_32.tsv
##     │   │   ├── up_33.tsv
##     │   │   ├── up_34.tsv
##     │   │   ├── up_35.tsv
##     │   │   ├── up_36.tsv
##     │   │   ├── up_37.tsv
##     │   │   ├── up_38.tsv
##     │   │   ├── up_39.tsv
##     │   │   ├── up_4.tsv
##     │   │   ├── up_40.tsv
##     │   │   ├── up_41.tsv
##     │   │   ├── up_5.tsv
##     │   │   ├── up_6.tsv
##     │   │   ├── up_7.tsv
##     │   │   ├── up_8.tsv
##     │   │   └── up_9.tsv
##     │   ├── logo
##     │   │   └── hexlogo.png
##     │   └── mapping
##     │       ├── handle_pmech.tsv
##     │       └── reference_set.tsv
##     ├── models
##     │   ├── dn_vs_lof_model.Rmd
##     │   ├── functions.R
##     │   ├── gof_vs_lof_model.Rmd
##     │   ├── lof_vs_nonlof_model.Rmd
##     │   └── model_data
##     │       ├── dn_imputed_full_frame_prep.tsv
##     │       ├── dom_class_2023-06.csv
##     │       ├── gene_to_uniprot_2023_02.tsv
##     │       ├── gof_imputed_full_frame_prep.tsv
##     │       ├── lof_imputed_full_frame_prep.tsv
##     │       ├── protNLM_2023_02.tsv.gz
##     │       ├── proteome_features.tsv
##     │       ├── seqmat.clustal
##     │       └── uniprot_2023_02.fa
##     └── supplement
##         ├── table_S1.csv
##         ├── table_S2.xlsx
##         └── table_S3.xlsx</code></pre>
<div id="readme-file-1" class="section level3 hasAnchor" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> README file<a href="guerrilla-analytics.html#readme-file-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Dit is de inhoud van de README file voor de data management van dit portfolio.</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="guerrilla-analytics.html#cb41-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">readLines</span>(<span class="st">&quot;README.md&quot;</span>), <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>)</span></code></pre></div>
<pre><code>## # DSFB2 Workflows Portfolio Jetske Moot       
##           
##           
## Welkom in de repository van het DSFB2 Workflows Portfolio van Jetske Moot.       
## *Hogeschool Utrecht, 2025*     
##       
##       
## ## Inhoud         
##         
##         
## Deze repository bevat een verzameling van de uitwerking van de portfolio opdrachten uitgevoerd tijdens de DSFB2 cursus. Het portfolio bestaat uit diverse R Markdown documenten, analyses, data sets en bijbehorende outputs, gestructureerd in mappen per opdracht of onderwerp.         
##           
##           
## ## Overzicht bestanden en mappen   
##         
##         
## | Bestandsnaam / Map                 | Korte omschrijving                                           |
## |----------------------------------|----------------------------------------------------------------|
## | 01_cv_jm.Rmd                     | Curriculum vitae (R Markdown document)                         |
## | 02_toekomst_jm.Rmd               | Plan voor de toekomst (R Markdown document)                    |
## | 03_vrije_opdracht_jm.Rmd         | Vrije opdracht (R Markdown document)                           |
## | 04_guerrilla_analytics.Rmd       | Guerrilla analytics opdracht (R Markdown document)             |
## | 05_reproducible.Rmd              | Reproduceerbare wetenschap (R Markdown document)               |
## | 06_rpackage.Rmd                  | Ontwikkeling van een R package (R Markdown document)           |
## | 07_projecticum.Rmd               | Introductie projecticum opdracht (R Markdown document)         |
## | 08_covid19.Rmd                   | Documentatie parameterized rapport COVID-19 data (R Mardown document) |
## | bibliography.json                | Bibliografische referenties                                    |
## | LICENSE                          | Licentie informatie                                            |
## | README.md                        | Huidige readme (wat nu wordt gelezen)                          |
## | dsfb2_workflows_portfolio_jm.Rproj | RStudio project                                              |
## | index.Rmd                       | Hoofdindex van het project (R Markdown document)                |
## | raw_data/                       | Map met ruwe data                                               |
## | images/                         | Map met afbeeldingen                                            |
## | _book/                          | Gebouwde HTML versie van het project (bookdown output)          |
## | _bookdown_files/                | Ondersteunende bestanden voor bookdown                          |
## | _main_files/figure-html/        | Figuurafbeeldingen gebruikt in het project                      |
## | daur2_eindopdracht/             | Submap met RNA-seq data en analyse                              |
## | repro_opdracht/                 | Submap met reproduceerbaarheid analyse                          |
## | protein_analysis                | Submap met code voor eiwitanalyse (vrije opdracht)              |
##           
##           
##            
## ## Licentie
##         
##         
## Deze repository is gelicenseerd onder de MIT License.       
##           
##           
## ## Contact
##     
##     
## Jetske Moot - GitHub Issues
## 
## 
## ----------------------------------------------------------------------------
##       
## Veel plezier met verkennen van mijn portfolio!</code></pre>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="structurele-eiwitanalyse-met-bio3d.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="peer-review.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
